firmware/Core/Inc/telecommands/i2c_telecommand_defs.h

```diff
@@ -0,0 +1,7 @@
+#include <stdint.h>
+#include "telecommands/telecommand_definitions.h"
+#include "main.h"
```

Comment:
This file needs an include guard (see other header files for examples).

Please include other header files from the `.c` files, and not from `.h` files.

Reply:
Done.